README.md

```diff
@@ -26,6 +26,31 @@ With a slight tweak of the y-axis label, core domain charts can be used to plan
 
 ![alt text](resources/architecture-migration-core-domain-chart.jpg "Architecture Migration Core Domain Chart")
 
+## Suggestions for Measuring Complexity and Differentiation
+
+Firstly, measuring complexity and differentiation of your domains is hard and is usually quite subjective. Strategy is a bet on the future, so you can never be sure exactly how complex or differentiating something will turn out to be. But there is still a lot of value in discussing and visualising your beliefs of the complexity and differentiation of each domain.
+
+The following clues can help to assess the complexity and differentiation of each domain. 
+ 
+ - How difficult is it to design a conceptual model for the domain and build (and maintain) it as software? (essential domain complexity)
+- Is the current technical solution more complex than it needs to be to provide the current functionality? (accidental technical complexity)
+- Are there complex processes, calculations, and decisions happening outside the software? (operational complexity)
+- How hard would it be for a new entrant to the market to match or exceed the capability?
+- How hard would it be for an existing competitor to match or exceed the capability?
+- How much of an advantage is currently produced by the domain (revenue, brand, engagement)?
+- How much of an advantage could be produced by domain (revenue, brand, engagement)?
+- What damage would occur to the brand if major or recurring failures happened in the domain?
+- Which cynefin domain does it fall within?
+
+Complexity can manifest in different forms. Here are some clues for uncovering the essential domain complexity, accidental technical complexity, and operational complexity of each (sub)domain:
+
+- the difficulty in discovering potential value
+- designing the rules, logic, and workflows are fundamentally hard to get right
+- simpler rules and logic but operating at large scale
+- high levels of differentiation are needed to displace existing market leaders or in saturated markets
+- specialist expertise that is difficult and expensive to acquire 
```

Comment:
Really think all of these are a great step @NTCoding 

I do miss more of these "team/org" dimension "clues" and "suggestions" here. At the moment it is very much focused on "domain" and "technical" complexity, which I think makes sense as they are the main focus in general on this exercise. However, I think adding that the teams/org perspective could be interesting when doing this sort of exercise, both looking at current domains but also "new domains" (/bets). Not yet 100% sure how to best capture that in a simple way, but the first thing that came to mind was using Team Topologies Cognitive Load ideas. For example: "domain complexity" may be simple, but given that the teams are not well supported organizationally (team topology wise, say: no platform or enabling teams supporting, or not well scoped teams), then the overall "complexity" surrounding that domain (and how to execute on it and evolve it) goes up... which means we may not be able to execute the strategy we are envisioning from a domain & technology perspective.

Reply:
@emgsilva I think you make a good point. Adding some organizational indicators of complexity could be useful. Maybe this blog post will help you trigger some thoughts: https://itrevolution.com/cognitive-load/

This list is just a starting point so anything we can add to it that other people will find useful is welcome.

Reply:
I think the list is helpful and they should be answered honestly. I also can relate to them and (unfortunately) tick some of them off :(